bin/amtctrl

```diff
@@ -61,6 +63,11 @@ def parse_args_rm():
                    help='')
     return parser.parse_args()
 
+def check_valid_vnc_password(passwd):
+    if re.search(r'^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8}\b$', passwd):
```

Comment:
I'd rather break this up into separate conditionals just to make it easier to understand later. I can do that post merge.